packages/server/postgres/migrations/1634351004998_addMattermostAuth.ts

```diff
@@ -0,0 +1,26 @@
+import {Client} from 'pg'
+import getPgConfig from '../getPgConfig'
+
+export async function up() {
+  const client = new Client(getPgConfig())
+  await client.connect()
+  await client.query(`
+  CREATE TABLE IF NOT EXISTS "MattermostAuth" (
+    "createdAt" TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP NOT NULL,
+    "updatedAt" TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP NOT NULL,
+    "isActive" BOOLEAN DEFAULT TRUE NOT NULL,
+    "webhookUrl" VARCHAR(2083) NOT NULL,
+    "userId" VARCHAR(100) NOT NULL,
+    "teamId" VARCHAR(100) NOT NULL,
+    PRIMARY KEY ("teamId")
```

Comment:
I think we should make this teamId, userId. 
me having the ability to remove your integration feels bad.